src/blocks/weather.rs

```diff
@@ -236,6 +252,8 @@ struct WeatherResult {
     location: String,
     current_weather: WeatherMoment,
     forecast: Option<Forecast>,
+    sunrise: DateTime<Utc>,
```

Comment:
Let's make sunrise/sunset optional so that if they aren't requested in the format string we don't have to fetch them